components/bookingButton/bookingButton.tsx

```diff
@@ -38,39 +28,23 @@ export const BookingButton = ({ data }) => {
   }
 
   const bookingPhone = layoutData.bookingPhone;
-
-  const showSuccessToast = () => {
-    toast.success(
-      <div id="success-toaster" className="text-left">
-        Form submitted. We&apos;ll be in contact as soon as possible.
-      </div>
-    );
-  };
+  const jotFormClass =
+    buttonClass ?? "mt-14" + " " + `lightbox-${layoutData.jotForm.id}`; // lightbox-id class is a trigger point for the JotForm IFrame
 
   return (
-    <div
-      className={twMerge("flex w-full flex-col items-center", containerClass)}
-    >
-      <UtilityButton
-        className={buttonClass || "mt-14"}
-        onClick={showBookingForm}
-        buttonText={buttonText}
-      />
-      {!hideCallUs && (
-        <h2 className="mx-auto max-w-full text-center">
-          or call us on {bookingPhone}
-        </h2>
-      )}
-      <Popup isVisible={isVisible} onClose={setIsVisible}>
-        {isVisible && (
-          <BookingForm
-            onClose={setIsVisible}
-            showSuccessToast={showSuccessToast}
-          />
+    <>
+      <JotFormScript />
+      <div
+        className={twMerge("flex w-full flex-col items-center", containerClass)}
+      >
+        <UtilityButton className={jotFormClass} buttonText={buttonText} />
```

Comment:
if it is just the class name that wires up the button to the class - better to have a jotform button component that abstracts that implementation detail away

Reply:
Fixed - added a component that can be reused